Geometry/CaloTopology/test/CaloTowerTopologyTester.cc

```diff
@@ -0,0 +1,82 @@
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESTransientHandle.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DetectorDescription/Core/interface/DDCompactView.h"
+#include "Geometry/Records/interface/IdealGeometryRecord.h"
+#include "Geometry/Records/interface/HcalRecNumberingRecord.h"
+#include "Geometry/HcalCommonData/interface/HcalDDDRecConstants.h"
+#include "Geometry/CaloTopology/interface/CaloTowerTopology.h"
+#include "DataFormats/CaloTowers/interface/CaloTowerDetId.h"
+
+class CaloTowerTopologyTester : public edm::EDAnalyzer {
```

Comment:
even though it's /test and note even in reco: should start using modern multithreading-aware modules, instead of legacy